autorest/codegen/models/list_schema.py

```diff
@@ -37,11 +37,11 @@ def get_validation_map(self):
         return validation_map or None
 
     @classmethod
-    def from_yaml(cls, yaml_data: Dict[str, str], **kwargs) -> "SequenceType":
+    def from_yaml(cls, yaml_data: Dict[str, str], **kwargs) -> "SequenceType":  # pylint: disable=arguments-differ
```

Comment:
Same here, if you remove kwargs in base schema, maybe we should remove it everywhere, or at it everywhere, but I really don't like a "arguments-differ" warning :(

Reply:
I got rid of the arguments-differ warning by adding kwargs to BaseSchema's from_yaml, and added a suppression of kwargs in BaseSchema's from_yaml

Reply:
who is using kwargs in all our subclasses?